### PR TITLE
Fix for parallel computation bug with od image computation.

### PR DESCRIPTION
**Pull request by hoytak**
This fixes the empty graph session error seen on machines with many cores.

### CODE REVIEW

File: src/python/turicreate/test/test_object_detector.py [hunk 1]
```diff
@@ -241,6 +241,12 @@ def test_predict(self):
         pred0 = self.model.predict(sf[:0])
         self.assertEqual(len(pred0), 0)
 
+    def test_predict_with_invalid_annotation(self):
```
Comment (srikris) [inline src/python/turicreate/test/test_object_detector.py]: This seems like a cherry picked commit from another branch. Bad merge?

Comment (jakesabathia2) [reply src/python/turicreate/test/test_object_detector.py]: hmmm it seems like this is picked up from my commit.

File: src/toolkits/object_detection/object_detector.cpp [hunk 2]
```diff
@@ -659,8 +659,11 @@ variant_type object_detector::predict(
   // Convert data to SFrame
   std::string image_column_name = read_state<flex_string>("feature");
   gl_sframe sframe_data = convert_types_to_sframe(data, image_column_name);
+  gl_sframe sframe_image_data(
```
Comment (jakesabathia2) [inline src/toolkits/object_detection/object_detector.cpp]: hmmm it seems like this is picked up from my commit too.

File: src/python/turicreate/test/test_object_detector.py (hunk 1 above)
Comment (hoytak) [reply src/python/turicreate/test/test_object_detector.py]: Quite possibly.  I squashed a bunch of trials down for the PR.  I'm happy to let you merge yours first.

Comment (jakesabathia2) [reply src/python/turicreate/test/test_object_detector.py]: I'm still running the gitlab for this PR.
I'll let you know if it is merged.

**Comment by jakesabathia2** [comment]
have merged my pr.

**Comment by srikris** [comment]
Closing this PR for now.

**Pull request closed without merging**